Formula/coccinelle.rb

```diff
@@ -22,10 +23,25 @@ class Coccinelle < Formula
   depends_on "autoconf" => :build
   depends_on "automake" => :build
   depends_on "hevea" => :build
+  depends_on "ocaml-findlib" => :build # for stdcompat resource
   depends_on "opam" => :build
   depends_on "ocaml"
 
+  # Bootstap resource for Ocaml 4.12 compatibility.
+  # Remove when Coccinelle supports Ocaml 4.12 natively
+  resource "stdcompat" do
+    url "https://github.com/thierry-martinez/stdcompat/releases/download/v15/stdcompat-15.tar.gz"
+    sha256 "5e746f68ffe451e7dabe9d961efeef36516b451f35a96e174b8f929a44599cf5"
+  end
+
   def install
+    resource("stdcompat").stage do
+      system "./configure", "--prefix=#{buildpath}/bootstrap"
+      ENV.deparallelize { system "make" }
+      system "make", "install"
+    end
+    ENV.prepend_path "OCAMLPATH", buildpath/"bootstrap"
```

Comment:
if this doesn't work this might need to be buildpath/bootstrap/lib instead, not just buildpath/bootstrap

Reply:
Yup, didn't seem to work. I'll wait to see how the Intel nodes fare and then give this a try.

However, I also noticed that `coccinelle` seems to be installing its own ocamlfindlib. Not sure why it's doing that, especially given that it appears to be installing the same version as Homebrew ocaml-findlib.